packages/make-hardener/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@agoric/make-hardener",
-  "version": "0.1.2+1-dev",
+  "version": "0.2.0",
```

Comment:
This should be 0.1.3 since there are no breaking changes. This package is consumed as `^0.1` and `0.2.0` won’t get picked up in an upgrade.

Reply:
You added a feature, so lerna is making a minor bump.